Workbooks/Network Insights/LoadBalancerWorkbooks/NetworkInsights-LoadBalancerMetrics/NetworkInsights-LoadBalancerMetrics.workbook

```diff
@@ -126,7 +124,7 @@
             "version": "KqlParameterItem/1.0",
             "name": "FrontendPublicIPsConstraint",
             "type": 1,
-            "value": "where false",
+            "value": "where id in~ ('/subscriptions/6bb4a28a-db84-4e8a-b1dc-fabf7bd9f0b2/resourcegroups/errobin/providers/microsoft.network/publicipaddresses/myip','/subscriptions/6bb4a28a-db84-4e8a-b1dc-fabf7bd9f0b2/resourcegroups/errobin/providers/microsoft.network/publicipaddresses/ip2')",
```

Comment:
Why this hardcoding?

Reply:
I'm not sure. I expected this to be removed when I set the Load Balancer id to come from query but it automatically populated this parameter with a default value. I've removed this and changed it back to false. I've tested and confirmed that the behavior is as expected with this field returned to false.

Reply:
@deveshjagwani  can you check how a false hardcoding works? If so, why we need a variable?

Reply:
mailed the details